gtk4/src/expression.rs

```diff
@@ -313,5 +379,16 @@ macro_rules! define_expression {
                 value
             }
         }
+
+        impl From<$rust_type> for glib::Value {
+            fn from(e: $rust_type) -> Self {
+                skip_assert_initialized!();
+                let mut value = glib::Value::for_value_type::<$rust_type>();
+                unsafe {
+                    ffi::gtk_value_set_expression(value.to_glib_none_mut().0, e.into_glib_ptr())
```

Comment:
This will leak the pointer

Reply:
Are you sure? This is using `into_glib_ptr` like the rest

Reply:
Yes, but it's `set_expression()` and not `take_expression()`

Reply:
Oh right :)